### PR TITLE
[Merged by Bors] - feat: port MeasureTheory.Function.SimpleFuncDenseLp

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by jcommelin** [comment]
This porting PR is hard to review. I don't know how to instruct github to show me a diff between mathport output and HEAD of the PR. In such cases, please use a combination of rebasing, squashing and force-pushing to create a nice rewritten history that is easy to review.

In this case, I've reviewed the individual commits, which is doable because there's an exceptionally finite number of them.

bors merge

**Review by eric-wieser — COMMENTED** [review]
@jcommelin, I believe [this](https://github.com/leanprover-community/mathlib4/pull/4521/files/bacabc8ee9eb9ae241078eb92f9850de05cd5f17..b2cc59bc266c87994edead78c76cdaf0351c95e6) is that diff

File: Mathlib/MeasureTheory/Function/SimpleFuncDenseLp.lean [hunk 1]
```diff
@@ -437,255 +437,254 @@ end SimpleFunc
...
+  add_mem' := by
+    rintro f g ⟨s, hs⟩ ⟨t, ht⟩
+    use s + t
+    simp only [← hs, ← ht, AEEqFun.mk_add_mk, AddSubgroup.coe_add, AEEqFun.mk_eq_mk,
+      SimpleFunc.coe_add]
+  neg_mem' := by
+    rintro f ⟨s, hs⟩
+    use -s
+    simp only [← hs, AEEqFun.neg_mk, SimpleFunc.coe_neg, AEEqFun.mk_eq_mk, AddSubgroup.coe_neg]
+#align measure_theory.Lp.simple_func MeasureTheory.Lp.simpleFunc
```
Comment (eric-wieser) [inline Mathlib/MeasureTheory/Function/SimpleFuncDenseLp.lean]: Why was this rewritten? Should this get a porting note?

Comment (Ruben-VandeVelde) [reply Mathlib/MeasureTheory/Function/SimpleFuncDenseLp.lean]: "unknown free variable '_uniq.546677'". I'll make a note after this PR is merged

**Pull request closed without merging**